package.json

```diff
@@ -2354,6 +2360,12 @@
                 "category": "%vscode-docker.commands.category.dockerImages%",
                 "icon": "$(clear-all)"
             },
+            {
+                "command": "vscode-docker.images.showDangling",
+                "title": "%vscode-docker.commands.images.showDangling%",
+                "category": "%vscode-docker.commands.category.dockerImages%",
+                "icon": "$(symbol-null)"
```

Comment:
A minor nit, but I wonder if either of the following icons might say "dangling images" more then `symbol-null`:

 - `outline-view-icon` 
<img width="26" alt="Screenshot 2021-05-27 at 09 36 26" src="https://user-images.githubusercontent.com/6402946/119863982-05381000-becf-11eb-9dfd-34aedebfaa29.png">

 - `testing-show-as-list-icon`

<img width="23" alt="Screenshot 2021-05-27 at 09 37 03" src="https://user-images.githubusercontent.com/6402946/119864068-1aad3a00-becf-11eb-8bc9-3a0cf537a86e.png">

Reply:
Yes, 
outline-view-icon is a good choice. I will commit this change.